src/main/java/frc/robot/subsystems/loader/LoaderSS.java

```diff
@@ -34,7 +34,7 @@ public void setDesiredVelocity(double desiredVelocity) {
      */
     @Override
     public void overriddenMove(double power) {
-        motor.set(power);
+        motor.setVoltage(power);
```

Comment:
Why isn't this giving an error? `overriddenMove` is inherited. if you change it it will complain that you don't have all inherited functions